sdk/network/Azure.ResourceManager.Network/src/Generated/Models/PacketCaptureCreateOrUpdateContent.cs

```diff
@@ -103,5 +107,9 @@ internal PacketCaptureCreateOrUpdateContent()
         public PacketCaptureStorageLocation StorageLocation { get; }
         /// <summary> A list of packet capture filters. </summary>
         public IList<PacketCaptureFilter> Filters { get; }
+        /// <summary> This continuous capture is a nullable boolean, which can hold 'null', 'true' or 'false' value. If we do not pass this parameter, it would be consider as 'null', default value is 'null'. </summary>
+        public bool? ContinuousCapture { get; set; }
```

Comment:
same

Reply:
Hi @archerzz , it's a nullable Boolean parameter, and we need this parameter in this format.